info.yml

```diff
@@ -126,6 +126,11 @@ configuration:
         default_value: True
         default_value_tk-hiero: False
 
+    auto_expand_tree:
+        type: bool
+        description: Automatically expands the root level of the treeviews.
+        default_value: True
```

Comment:
OK I have set it False now. Rob says it was never intended to be an option, I can remove the option if we feel that is best?